docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,256 @@
+###############################################
+Compute an expectation value with ``Estimator``
+###############################################
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the :class:`~qiskit.primitives.Estimator` primitive.
+
+.. note::
+
+    While this guide only uses Qiskit Terra's implementation of the ``Estimator`` primitive, there are other
+    implementations of this primitive like Qiskit Terra's :class:`~qiskit.primitives.BackendEstimator`, Qiskit Aer's :class:`~qiskit_aer.primitives.Estimator`
+    and Qiskit Runtime's :class:`~qiskit_ibm_runtime.Estimator`.
+
+
+Initialize observable
+=====================
+
+The first step is to define the observable whose expected value you want to compute. This observable can be any ``BaseOperator``, like the operators from :mod:`qiskit.quantum_info`.
```

Comment:
There's actually no documentation for `qiskit.quantum_info.operators.base_operator.BaseOperator` 😅 (at least that I could find)

I already tried writing ```:class:`~.BaseOperator` ```, ```:class:`~qiskit.quantum_info.operators.base_operator.BaseOperator` ``` and even ```:class:`~qiskit.quantum_info.BaseOperator` ``` but none of them generated any link.

Moreover if you check, for example, [the page of the Clifford operator](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Clifford.html) you'll see that `BaseOperator` appears in "Bases" but there's no link either.

Reply:
However, even though there would be no actual link if I wrote ```:class:`~.BaseOperator` ``` or  ```:class:`~qiskit.quantum_info.operators.base_operator.BaseOperator` ``` the output in the guide would look the same as if I wrote ``` ``BaseOperator`` ```. 

Would it be better if I wrote the Sphinx link in case we eventually decide to document that class?